__tests__/app.test.js

```diff
@@ -57,11 +62,35 @@ describe('GET /api/topics', () => {
         });
       });
   });
+});
 
-  it('404: responds with a 404 status for endpoints that do not exist on /api', () => {
+describe('GET /api/articles', () => {
+  it('200: responds with all articles each of which include the required properties sorted by date in descending order', () => {
     return request(app)
-      .get('/api/bananas')
-      .expect(404);
+      .get('/api/articles')
+      .expect(200)
+      .then((response) => {
+        const { articles } = response.body
+
+        expect(articles).toBeInstanceOf(Array);
+        expect(articles).toHaveLength(13);
+        expect(articles).toBeSortedBy('created_at', { descending: true });
```

Comment:
Awesome use of the jest sorted package here to check the default ordering of responses.